pgmoon/json.moon

```diff
@@ -0,0 +1,10 @@
+encode_json = (tbl) ->
+  json = require "cjson"
+  enc = json.encode tbl
+  "'#{(enc\gsub "'", "''")}'"
```

Comment:
I would have liked that too but since `escape_literal` belongs to a `pg` instance I am not really sure how to achieve this.